config/initializers/constants.rb

```diff
@@ -2,11 +2,3 @@
 MAX_NUMBER_LINKS_FUNDER = 5
```

Comment:
These could probably be moved elsewhere too. Maybe the branding.yml would be a good candidate